react-native-libraries.json

```
@@ -9506,5 +9506,18 @@
     "ios": true,
     "android": true,
     "expo": true
-  }
+  },
+  {
+  "githubUrl": "https://github.com/AtharvaDeolalikar/react-native-animated-bottom-drawer",
+  "npmPkg": "react-native-animated-bottom-drawer",
+  "nameOverride": "react-native-animated-bottom-sheet",
+  "ios": true,
+  "android": true,
+  "web": false,
+  "expo": true,
+  "unmaintained": false,
+  "dev": false,
+  "template": false,
+  "newArchitecture": false
```

Hello @AtharvaDeolalikar, thank you for adding new entry to the directory! 👍 

I have few suggestion for the new entry:

```suggestion
  "githubUrl": "https://github.com/AtharvaDeolalikar/react-native-animated-bottom-drawer",
  "ios": true,
  "android": true
```

`npmPkg` can be skipped, since package name matches the repository. I would also recommend to skip `nameOverride`, especially because you have used other package named in there, which can confuse the directory users. Last nit is that all `false` flags can be skipped, this is mostly for the readability and to reduce JJSON file size.